### DISCUSSION TITLE
Talks regarding this subject

### DISCUSSION THREAD

**Discussion by Tavh** [zakirullin/cognitive-load#18] [Show and tell]
I'm actually giving talks about learning in several confrences, i'll make sure to add a link to this repo in the next instance 👍

**Comment by zakirullin** [comment]
@Tavh thank you so much! 

Please share the talks once you hold them!

**Reply by Tavh** [reply]
I'm giving one tomorrow at Data Saturday Edinburgh 😁

If you're interested, this is a link to my previous one at NDC Oslo:

https://www.youtube.com/watch?v=El-YLgp9jqA

Starts at around 9:15:30

**Reply by TomCools** [reply]
Some of these slides look familiar. 🤣

**Reply by Tavh** [reply]
@TomCools 
Hey so cool that you saw it!
Notice the references list at the opening slide.

I wonder if you remember me, I was the guy that shared the game engine experience in your Devoxx 2022 talk. Also asked you about delievering a similar talk right after.
It was definetly the most memorable talk for me

**Reply by TomCools** [reply]
I remember you. :) Thanks for the reference. I'm happy you are also sharing the message and doing your own thing with it. It's a first for me to have someone mention me and use some of the content I talked about in a talk of their own. 👍  Best of luck on your next couple of talks.

For reference, this is the link to my latest recording: https://www.youtube.com/watch?v=nwhwNfO1WMQ

**Reply by TomCools** [reply]
Also. This is the link to your talk... cut out from the long one. https://youtu.be/msNjvbKlWRE
